public/ticketmanager.php

```diff
@@ -742,17 +742,14 @@
                     echo "<select name='action' required>";
                     echo "<option value='' disabled selected hidden>Choose an action...</option>";
                     if ($reportState == 1) {
-                        if ($user == $reportedBy) { // only the reporter can close as a mistaken report
-                            echo "<option value='closed-mistaken'>Close - Mistaken report</option>";
-                        }
```

Comment:
Shouldn't the reporter still be able to select this option, even if they're not a dev?

Reply:
That's a good catch.